packages/amazonq/test/e2e_new/amazonq/utils/generalUtils.ts

```diff
@@ -104,6 +192,19 @@ export async function clearChat(webview: WebviewView): Promise<boolean> {
     }
 }
 
+/**
+ * Creates a new text file and returns the editor
+ * @param workbench The Workbench instance
+ * @returns Promise<TextEditor> The text editor for the new file
+ */
+export async function createNewTextFile(workbench: Workbench, editorView: EditorView): Promise<TextEditor> {
+    await workbench.executeCommand('Create: New File...')
```

Comment:
If these command prompts are static, it might be worth factoring them out to a constants file.